doc/classes/Node.xml

```diff
@@ -1018,7 +1020,7 @@
 			Similar to [member process_priority] but for [constant NOTIFICATION_PHYSICS_PROCESS], [method _physics_process] or the internal version.
 		</member>
 		<member name="process_priority" type="int" setter="set_process_priority" getter="get_process_priority" default="0">
-			The node's execution order of the process callbacks ([method _process], [method _physics_process], and internal processing). Nodes whose priority value is [i]lower[/i] call their process callbacks first, regardless of tree order.
+			The node's execution order of the process callback ([method _process], [constant NOTIFICATION_PROCESS]). Nodes whose priority value is [i]lower[/i] call their process callback first, regardless of tree order.
```

Comment:
This removes internal processing which is incorrect, they are also affected by priority

Reply:
I added back the "and internal processing"